Source/WebInspectorUI/UserInterface/Views/DataGrid.css

```diff
@@ -49,7 +50,7 @@
 
 .data-grid .data-container {
     position: absolute;
-    top: 23px;
+    top: calc(var(--data-grid-header-height) + var(--data-grid-header-border-size));
```

Comment:
This didn't need fixing, but `23px` was a "magical number". This makes it explicit.